recipes-bsp/u-boot/u-boot-tools_2024.04.bb

```diff
@@ -1,4 +1,4 @@
 require recipes-bsp/u-boot/u-boot.inc
 require recipes-bsp/u-boot/u-boot-common.inc
-
-SRCREV = "13c1100335e40acb1066e074eb061387fd103c36"
+SRC_URI = "git://source.denx.de/u-boot/u-boot.git;protocol=https;branch=rpi5-2024.04-xt"
```

Comment:
I am not sure that DENX repo has rpi-2024.04-xt branch. How you tested this?

Reply:
dropped this commit and will create a separate PR as was requested by @GrygiriiS